# Service to get Aruco Pose
int64 tag_id
---
float32 x
float32 y
float32 z
float32 q0
float32 qx
float32 qy
float32 qz
